src/components/Common/View.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ComponentType } from "react";
+
+import { useView } from "@/Utils/useView";
+
+export default function View({
+  name,
+  board,
+  list,
+}: {
+  name: string;
+  board: ComponentType<{ setView: (view: string) => void }>;
+  list: ComponentType<{ setView: (view: string) => void }>;
+}) {
+  const [view, setView] = useView(`${name}DefaultView`);
```

Comment:
why complicate this?

```suggestion
  const [view, setView] = useView(name);
```